tensorflow/contrib/distributions/python/kernel_tests/bijectors/reshape_test.py

```diff
@@ -150,6 +151,17 @@ def _testInvalidDimensionsOpError(self, expected_error_message):
       with self.assertRaisesError(expected_error_message):
         sess.run(bijector.forward_event_shape_tensor(shape_in),
                  feed_dict=feed_dict)
+
+  def _testInvalidDimensionsStatic(self, expected_error_message):
+    """Version of _testInvalidDimensionsOpError for errors detected statically
+    at graph construction time."""
+    shape_in, shape_out, feed_dict = self.build_shapes([2, 3], [1, 2, -2,])
+    with self.assertRaisesWithPredicateMatch(errors.InvalidArgumentError,
+                                             expected_error_message):
+      bijector = Reshape(
```

Comment:
Unused variable 'feed_dict' [unused-variable]

Reply:
Changed `feed_dict` to `_`.